### DISCUSSION TITLE
Weird behavior of attached pictures

### DISCUSSION THREAD

**Discussion by SimonBlancoE** [phase1geo/Minder#316] [Q&A]
I'm not sure why is this happening, here's the description:

I attach a picture to a node.
I close the right sidebar pressing ESC
I select a different node and press shift+e to start editing the node info
The same image appears as attached in the "Current" tab (although it's not)

This only happens if you close the sidebar (either with ESC or clicking the icon), move to another node, and open the sidebar again. 
Clicking outside and back into the node solves the problem and shows the correct info. However, it is a bit confusing and it becomes a hurdle when using shortcuts mainly. 

I am using Minder 1.11.3-1 on Manjaro Gnome. I am very new in github, I'm not sure if I should open an issue about this.

**Comment by phase1geo** [comment]
Sounds like a bug. I’ve converted this into an issue for you.
